setup.txt

```diff
@@ -0,0 +1,12 @@
+#Softwares needed on local to install VAYU
```

Comment:
Please rename this file as well, there is a python convention of a file called `setup.py` which uses `setuptools`, which can easily install all dependencies of the project.